src/Main.java

```diff
@@ -1,5 +1,98 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Привет, мир!");
+        // задача №1
+        int a = 1000000;
+        System.out.println("Значение пременной a " + " с типом int " + " равно " + a);
```

Comment:
```suggestion
        System.out.println("Значение пременной a с типом int равно " + a);
```
зачем разделять строки? можно вот так было сделать